resource/labeling-app/components/category-selector.js

```diff
@@ -205,7 +205,7 @@ let CategorySelector = {
                     </div>
                     <div v-else style="width: 100%;">
                         <h2 v-if="status !== 'SKIPPED'">No labels yet</h2>
-                        <p v-if="type === 'image-object'" style="margin: auto;">Select a category by ...</p>
+                        <p v-if="isMultiLabel" style="margin: auto;">Select a category by ...</p>
                         <div class="circles-container">
                             <div style="max-width: 110px;">
                                 <div class="circle cat-example"></div>
```

Comment:
I think we're still missing "or" between bubbles
![image](https://user-images.githubusercontent.com/7889985/101198182-59baab00-3663-11eb-9060-0d620492c0b1.png)

Reply:
Fixed in af87c6a